packages/makedumpfile/0000-fix-strip-invocation-for-TARGET-env-variable.patch

```diff
@@ -0,0 +1,25 @@
+From 20485e6cbd2b6e61fba04c0687a05053a07b3131 Mon Sep 17 00:00:00 2001
+From: Arnaldo Garcia Rincon <agarrcia@amazon.com>
+Date: Thu, 27 May 2021 19:02:52 +0000
+Subject: [PATCH] fix strip invocation for TARGET env variable
+
+---
+ Makefile | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/Makefile b/Makefile
+index 6fa00bc..f82c10a 100644
+--- a/Makefile
++++ b/Makefile
+@@ -17,7 +17,7 @@ CFLAGS_ARCH := $(CFLAGS_BASE)
+ HOST_ARCH := $(shell uname -m)
+ # Use TARGET as the target architecture if specified.
+ # Defaults to uname -m
+-ifeq ($(strip($TARGET)),)
++ifeq ($(strip $TARGET),)
```

Comment:
I am pretty sure the stripped value ends up as "ARGET" meaning this would always be false.

```suggestion
+ifeq ($(strip $(TARGET)),)
```